src/bindings/python/src/openvino/frontend/tensorflow/utils.py

```diff
@@ -7,7 +7,7 @@
 
 import logging as log
 import sys
-from distutils.version import LooseVersion
+from packaging.version import parse, Version
```

Comment:
it's not a standard python module and requires adding a new dependency. is it possible to avoid?

Reply:
It's not standard one but it's originating from pypa, the same authors as pip/setuptools/wheel packages. If the avoidance is not possible or require a lot of custom solutions -- I will vote for keeping it.

@akuporos Am I right that you are targeting https://github.com/pypa/packaging ? Please also add it to requirements for wheels if the package stays as a solution.

Reply:
discussed offline with @ilya-lavrenov